app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -39,6 +39,24 @@ def create
     redirect_to edit_chat_channel_membership_path(membership)
   end
 
+  def join_channel
+    membership_params = params[:chat_channel_membership]
+    chat_channel = ChatChannel.find(membership_params[:chat_channel_id])
+    user = User.find_by(id: membership_params[:user_id])
```

Comment:
Can we use `current_user` here instead to avoid someone being able to send any user_id they want?

Reply:
Yes, we can do that. Modified.
Thanks